### PR TITLE
tidb-operator: fix documentation usability issues in GCP document

### PR DESCRIPTION
**Pull request by yikeke**
### What problem does this PR solve? 

fix documentation usability issues found in GCP deployment document

### Check List 

Tests 

 - Unit test
 - E2E test
 - Stability test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Helm charts change
 - Has Go code change
 - Has CI related scripts change
 - Has documents change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:
 
 ```release-note

 ```

### CODE REVIEW

File: deploy/gcp/README.md [hunk 1]
```diff
@@ -53,13 +64,17 @@ terraform init
...
+
 After `terraform apply` is successful, the TiDB cluster can be accessed by SSHing into the bastion machine and connecting via MySQL:
+
 ```bash
 gcloud compute ssh bastion --zone <zone>
 mysql -h <tidb_ilb_ip> -P 4000 -u root
 ```
 
-It is possible to interact with the cluster using `kubectl` and `helm` with the kubeconfig file `credentials/kubeconfig_<cluster_name>`. The default `cluster_name` is `my-cluster`, it can be changed in `variables.tf`
+It is possible to interact with the cluster using `kubectl` and `helm` with the kubeconfig file `credentials/kubeconfig_<cluster_name>`. The default `cluster_name` is `my-cluster`, it can be changed in `variables.tf`:
```
Comment (DanielZhangQD) [inline deploy/gcp/README.md]: 下面到upgrade之前不是access database了，可以再起一个section，比如叫interact with the cluster 或者manage the cluster?

File: deploy/gcp/README.md [hunk 2]
```diff
@@ -98,29 +110,43 @@ GCP allows attaching a local SSD to any instance type that is `n1-standard-1` or
...
 $ gcloud compute instance-groups managed delete-instances gke-my-cluster-monitor-pool-08578e18-grp --instances=gke-my-cluster-monitor-pool-08578e18-c7vd --zone us-west1-b
-```
\ No newline at end of file
+```
+
+## Destroy
+
+When you are done, the infrastructure can be torn down by running:
+
+``` shell
```
Comment (DanielZhangQD) [inline deploy/gcp/README.md]: 上面都是用的bash，这里需要统一一下吗？

File: deploy/gcp/README.md [hunk 3]
```diff
@@ -44,7 +32,41 @@ gcloud services enable compute.googleapis.com && \
...
+
+* `TF_VAR_GCP_CREDENTIALS_PATH`: Path to a valid GCP credentials file. 
+    - It is recommended to create a new service account to be used by Terraform. See [this page](https://cloud.google.com/iam/docs/creating-managing-service-accounts) to create a service account and grant `Project Editor` role to it. 
+    - See [this page](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) to create service account keys, and choose `JSON` key type during creation. The downloaded `JSON` file that contains the private key is the credentials file you need.
+* `TF_VAR_GCP_REGION`: The region to create the resources in, for example: `us-west1`.
+* `TF_VAR_GCP_PROJECT`: The name of the GCP project.
+
+> *Note*: The service account must have sufficient permissions to create resources in the project. The `Project Editor` primitive will accomplish this.
+
+To set the three environment variables, you can first run `vi ~/.bash_profile` and insert the `export` statements in it. Here is an example in `~/.bash_profile`:
```
Comment (DanielZhangQD) [inline deploy/gcp/README.md]: Suggest to replace "insert" with "append", BTW, also need to source the file (source ~/.bash_profile or . ~/.bash_profile) after editing

File: deploy/gcp/README.md [hunk 4]
```diff
@@ -71,25 +124,41 @@ kubectl get po -n tidb
...
+For example, to scale out the cluster, you can modify the number of TiDB instances from 2 to 3:
+
+```
+variable "tidb_count" {
+default = 3
+}
+```
 
-> *Note*: Incrementing the node count will create a node per GCP availability zones.
+> *Note*: Incrementing the node count will create a node per GCP availability zone.
```
Comment (DanielZhangQD) [inline deploy/gcp/README.md]: Should be "available zone"?

**Comment by yikeke** [comment]
I have updated some examples and descriptions according to other deployment docs (see [PR#507](https://github.com/pingcap/tidb-operator/pull/507)). PTAL @DanielZhangQD @jlerche

File: deploy/gcp/README.md (hunk 4 above)
Comment (yikeke) [reply deploy/gcp/README.md]: In the `variables.tf` file, the descriptions for `tidb_count`, `tikv_count` and other variables all use "per GCP availability zone", so I think we should unify the wording by using "availability" here.

File: deploy/gcp/README.md [hunk 5]
```diff
@@ -107,76 +109,91 @@ gcloud compute ssh bastion --zone <zone>
...
+  description = "TiDB version"
+  default     = "v2.1.10"
 }
 ```
 
-The upgrading does not finish immediately. You can watch the upgrading process by `kubectl --kubeconfig credentials/kubeconfig_<cluster_name> get po -n tidb --watch`.
+The upgrading does not finish immediately. You can run `kubectl --kubeconfig credentials/kubeconfig_<cluster_name> get po -n tidb --watch` to verify that all pods are in `Running` state. Then you can [access the database](#access-the-database) and use `tidb_version()` to see whether the cluster has been upgraded successfully:
+
+```sql
+MySQL [(none)]> select tidb_version()\G
```
Comment (DanielZhangQD) [inline deploy/gcp/README.md]: Is "\G" required?

File: deploy/gcp/README.md [hunk 6]
```diff
@@ -44,7 +32,43 @@ gcloud services enable compute.googleapis.com && \
...
+* `TF_VAR_GCP_CREDENTIALS_PATH`: Path to a valid GCP credentials file. 
+    - It is recommended to create a new service account to be used by Terraform. See [this page](https://cloud.google.com/iam/docs/creating-managing-service-accounts) to create a service account and grant `Project Editor` role to it. 
+    - See [this page](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) to create service account keys, and choose `JSON` key type during creation. The downloaded `JSON` file that contains the private key is the credentials file you need.
+* `TF_VAR_GCP_REGION`: The region to create the resources in, for example: `us-west1`.
+* `TF_VAR_GCP_PROJECT`: The name of the GCP project.
+
+> *Note*: The service account must have sufficient permissions to create resources in the project. The `Project Editor` primitive will accomplish this.
+
+To set the three environment variables, you can first run `vi ~/.bash_profile`, append the `export` statements to it and run `source ~/.bash_profile`. 
+
```
Comment (jlerche) [inline deploy/gcp/README.md]: You can actually type `export FOO=var` or something like that in the terminal, so setting the environment variables in bash_profile is a little overkill. I would suggest removing the line "To set the three environment variables, you can first run `vi ~/.bash_profile`, append the `export` statements to it and run `source ~/.bash_profile`. "

File: deploy/gcp/README.md [hunk 7]
```diff
@@ -44,7 +32,43 @@ gcloud services enable compute.googleapis.com && \
...
+Here is an example of `~/.bash_profile`:
```
Comment (jlerche) [inline deploy/gcp/README.md]:
```suggestion
For example, in your terminal you can enter:
```

Comment (yikeke) [reply deploy/gcp/README.md]: Updated. PTAL again. @jlerche @DanielZhangQD

File: deploy/gcp/README.md (hunk 6 above)
Comment (DanielZhangQD) [reply deploy/gcp/README.md]: If export in the terminal, after customer quit from the session and then open another session to run terraform, the variables need to be exported again, I think it's not convenient

**Pull request merged**